templates/pages/admin/user/user.html.twig

```diff
@@ -55,7 +55,7 @@
                 <div class="d-flex flex-row flex-wrap flex-xl-nowrap">
                     <div class="row flex-row align-items-start flex-grow-1">
                         <div class="row flex-row">
-                            <input type="hidden" name="id" value="{{ session('glpiID') }}" />
+                            <input type="hidden" name="id" value="{{ is_preference_form ? session('glpiID') : item.getID() }}" />
```

Comment:
I guess it would be better to not send `id` field at all when creating a new user.